.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,35 @@
+name: 🚀 Feature request
+description: Suggest an idea for this project.
+labels: [enhancement]
```

Comment:
This adds the label enhancement to each feature request to enable simple filtering for bugs or feature request.